routers/users.py

```
@@ -0,0 +1,93 @@
+    existing_user = data.find_by_field("users.json", "email", user_data.email)
+    if existing_user:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="이미 등록된 이메일"
+        )
+
+    hashed_password = auth.hash_password(user_data.password)
+
+    user_id = data.get_next_id("users.json")
+
+    now = datetime.now().isoformat()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)